Live-Snap/Live-Snap/Controllers/FetchSnapcodeViewController.swift

```diff
@@ -126,7 +126,7 @@ class FetchSnapcodeViewController: UIViewController, UITextFieldDelegate {
         print("Invalid snapchat username")
     }
     
-    func sanitizeSnapchatUsernameString(username: String) -> String? {
+    func sanitizeSnapchatUsernameString(username: String?) -> String? {
```

Comment:
why would this take an optional? no point in that since this function is useless if the passed in username is nil